CRM/ManualDirectDebit/Mail/DataCollector/Base.php

```diff
@@ -371,4 +371,16 @@ private function getDdCode($dd_code) {
     return $result['values'][0]['api.OptionValue.getValue'];
   }
 
+  /**
+   * Converts date time format
+   *
+   * @param $date
+   *
+   * @return string
+   */
+  private function convertDateFormat($date) {
+    $myDateTime = DateTime::createFromFormat('Y-m-d H:i:s', $date);
+    return $myDateTime->format('d/m/Y');
```

Comment:
There is  CiviCRM core class that helps deal with dates. I'm not sure of what the actual use case is here, but I would think you'd need to format the date according to the current user's preferred format. To do this, you can do:

```php
private function convertDateFormat($date) {
  return CRM_Utils_Date::customFormat($date);
}
```

If that format is mandatory though, you can pass it as the second parameter:
```php
private function convertDateFormat($date) {
  return CRM_Utils_Date::customFormat($date, '%d/%m/%Y');
}
```